mungegithub/mungers/release-note-label.go

```diff
@@ -125,8 +130,21 @@ func (r *ReleaseNoteLabel) Munge(obj *github.MungeObject) {
 		return
 	}
 
-	if !obj.HasLabel(releaseNoteLabelNeeded) {
-		obj.AddLabel(releaseNoteLabelNeeded)
+	labelToAdd := determineReleaseNoteLabel(obj)
+	if labelToAdd == releaseNoteLabelNeeded {
+		if !obj.HasLabel(releaseNoteLabelNeeded) {
+			obj.AddLabel(releaseNoteLabelNeeded)
+		}
+		if !obj.HasLabel(doNotMergeLabel) {
+			obj.AddLabel(doNotMergeLabel)
+		}
+	} else {
+		//going to apply some other release-note-label
+		if obj.HasLabel(releaseNoteLabelNeeded) {
+			obj.RemoveLabel(releaseNoteLabelNeeded)
+		}
+		obj.AddLabel(labelToAdd)
```

Comment:
oops.. sorry, never saw this. Yeah, it is redundant as far as I know, because the submit-queue gates on no-lgtm and no-release-note anyway.